projects/SD/test/README

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
wtf is this readme... it can stay tho

Reply:
<img width="486" alt="brtk_rdme_po_tygodniu" src="https://github.com/grupacosmo/rocket-playground/assets/116205604/66d2b1b5-9b94-497f-9951-5a948b5a4347">

Reply:
1 week later